BlobStorage/Advanced.cs

```diff
@@ -514,26 +409,26 @@ private static void PrintServiceClientProperties(CloudBlobClient blobClient)
         /// <summary>
         /// Creates a sample container for use in the sample application.
         /// </summary>
-        /// <param name="blobClient">The blob service client.</param>
-        /// <returns>A CloudBlobContainer object.</returns>
-        private static async Task<CloudBlobContainer> CreateSampleContainerAsync(CloudBlobClient blobClient)
+        /// <param name="blobServiceClient">The blob service client.</param>
+        /// <returns>A BlobContainerClient object.</returns>
+        private static async Task<BlobContainerClient> CreateSampleContainerAsync(BlobServiceClient blobServiceClient)
         {
             // Name the sample container based on new GUID, to ensure uniqueness.
             string containerName = ContainerPrefix + Guid.NewGuid();
 
             // Get a reference to a sample container.
-            CloudBlobContainer container = blobClient.GetContainerReference(containerName);
+            BlobContainerClient container = blobServiceClient.GetBlobContainerClient(containerName);
 
             try
             {
                 // Create the container if it does not already exist.
                 await container.CreateIfNotExistsAsync();
             }
-            catch (StorageException e)
+            catch (RequestFailedException e)
             {
-                // Ensure that the storage emulator is running if using emulator connection string.
+                // Ensure that the azurite is running if using azurite connection string.
                 Console.WriteLine(e.Message);
-                Console.WriteLine("If you are running with the default connection string, please make sure you have started the storage emulator. Press the Windows key and type Azure Storage to select and run it from the list of applications - then restart the sample.");
+                Console.WriteLine("If you are running with the default connection string, please make sure you have started the Azurite. Press the Windows key and type Azure Storage to select and run it from the list of applications - then restart the sample.");
```

Comment:
Is this 2nd sentence still accurate? It might be best to instead link to https://docs.microsoft.com/azure/storage/common/storage-use-azurite.

```suggestion
                Console.WriteLine("If you're running with the default connection string, make sure you've started the Azurite emulator. Press the Windows key and type Azure Storage to select and run it from the list of apps. Then restart the sample.");
```